js/pnpm-lock.yaml

```diff
@@ -14,6 +14,9 @@ importers:
       '@mantine/hooks':
         specifier: 7.10.2
         version: 7.10.2(react@18.3.1)
+      '@tabler/icons':
```

Comment:
You need to remove this change from the commit too, since you've removed the package.json changes